dspace-api/src/main/java/org/dspace/discovery/configuration/DiscoverySearchFilterFacet.java

```diff
@@ -55,4 +55,12 @@ public void setSortOrderSidebar(DiscoveryConfigurationParameters.SORT sortOrderS
     public String getFilterType() {
         return FILTER_TYPE_FACET;
     }
+
+    public boolean isExposeMinMax() {
```

Comment:
Could we add Javadocs here? I'm not sure I understand what "MinMax"is referring to.